pages/content/amp-dev/documentation/courses/beginning-course/our-first-amp-page@pt_BR.md

```diff
@@ -0,0 +1,451 @@
+
+### Solução
+
+A solução pode ser encontrada nesse <a href="https://glitch.com/~hungry-modem" target="_blank">examplo do Glitch</a>. A parte da página que contém as alterações deve ser semelhante a esta:
+
+[sourcecode:html]
+{% raw %}<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>Chico's Cheese Bicycles</title>
+  <link rel="shortcut icon" href="https://cdn.glitch.com/d7f46a57-0ca4-4cca-ab0f-69068dec6631%2Fcheese-favicon.png?1540228964214">
+  <meta name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link rel="canonical" href="https://hungry-modem.glitch.me/">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    ... styles elided due to length ...
+  </style>
+</head>
+{% endraw %}[/sourcecode]
+
+## A Valid AMP Page...Almost
+
+Se você completou todos os exercícios acima, parabéns! Sua página é quase uma página AMP válida. Se você abrir sua página e o Chrome DevTools (`Command + Option + I` no Mac ou` Control + Shift + I` no Windows / Linux) e selecionar a guia **Console**, deverá ver esta mensagem no console , indicando que a biblioteca AMP foi carregada com sucesso:
+
+```
+Powered by AMP ⚡ HTML
+```
+
+Em seguida, se você abrir a extensão do Validador de AMP, isso mostra que chegamos ao último erro:
+
+```
+The tag 'img' may only appear as a descendant of tag 'noscript'. Did you mean 'amp-img'?
+```
+
+Este é um erro importante de entender. Algumas tags HTML não são permitidas em documentos AMP. Em alguns casos, o AMP exige que você use uma alternativa. Chamamos essas tags HTML personalizadas e não padrão de "componentes" e iremos discuti-las com mais detalhes posteriormente na próxima seção deste treinamento
```

Comment:
Very minor point: the last sentence is missing the final period.

(Get it? "Point"?)

Reply:
Done!